server/tests/github.test.ts

```diff
@@ -0,0 +1,166 @@
+import { resetBranch, getRepo, fetchRepos } from '../src/github'
+import { Octokit } from 'probot'
+
+describe('github:', () => {
+  let github: Octokit
+  const owner = 'maticzav'
+  const repo = 'emma-cli'
+
+  describe('resetBranch:', () => {
+    let getBranch: jest.Mock
+    let deleteRef: jest.Mock
+    let getRef: jest.Mock
+    let createRef: jest.Mock
+
+    const branchName = 'test-branch'
+    const defaultBranchName = 'default-branch'
+
+    beforeEach(() => {
+      github = new Octokit()
```

Comment:
I think it's easier to have a single object and supply leafs with `jest.fn()` than nesting a path for each of them.